ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -158,209 +161,227 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
-                                                                    <div className='columns small-3'>Date:</div>
-                                                                    <div className='columns small-9'>
-                                                                        <Timestamp date={metadata.date} />
-                                                                    </div>
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Tags:</div>
-                                                                    <div className='columns small-9'>
-                                                                        {((metadata.tags || []).length > 0 && metadata.tags.join(', ')) || 'No tags'}
                                                                     </div>
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Author:</div>
-                                                                    <div className='columns small-9'>{metadata.author}</div>
-                                                                </div>
-                                                            </div>
-                                                            <div className='white-box__details'>
-                                                                <div className='row white-box__details-row'>
-                                                                    <div className='columns small-3'>Message:</div>
-                                                                    <div className='columns small-9' style={{display: 'flex', alignItems: 'center'}}>
-                                                                        <div className='application-details__commit-message'>{renderCommitMessage(metadata.message)}</div>
+                                                                </React.Fragment>
+                                                            )
+                                                        }}>
+                                                        <div className='application-details__application-search-bar'>
```

Comment:
I wonder if the search bar should be in the same position as in the app list page, eg. right next to the Refresh button at the top.

Reply:
I will be really hard to implement it that way because those buttons are in a different component in argo-ui. I talked with Alex about this and he suggested to keep it with `application-details.tsx`.

Reply:
Ok, as discussed, because I have a widescreen monitor and I have made the browser wide in width, everything would just fit nicely.

<img width="1774" alt="width" src="https://user-images.githubusercontent.com/7726022/145892640-2c57ff25-be2a-4fdc-b067-fb92be5134c9.png">